client/components/LearnerViewDirective.js

```diff
@@ -584,12 +581,7 @@ tie.directive('learnerView', [function() {
             var syntaxErrorIndex = feedback.getSyntaxErrorIndex();
```

Comment:
Mm for the reinforcement strings to show up, we rely on knowing whether or not there was a syntax error. So I left it because although we don't specifically need the index, the fact that if there isn't an index shows that there wasn't a syntax error. Should I change it to a boolean instead?